cmd/awscollector/Dockerfile

```diff
@@ -10,12 +10,14 @@ RUN apk --update add ca-certificates
 WORKDIR /workspace
 
 ARG TARGETPLATFORM
-RUN echo "Targate platform is $TARGETPLATFORM"
+RUN echo "Target platform is $TARGETPLATFORM"
 
 # copy artifacts
 # always assume binary is created
 COPY build/linux/* /workspace/
-COPY cmd/awscollector/copy_docker_binary.sh  /workspace/cmd/awscollector/copy_docker_binary.sh
+COPY config.yaml /workspace/config/otel-config.yaml
+COPY config/ /workspace/config/
```

Comment:
Any reason why we should not copy directly but instead copy twice?

Reply:
It's mainly to reduce the number of COPY layers in the actual image to just one.

Reply:
Reasonable.